Adding missing tutorial abstracts

mjpost opened the pull request:
Here we are adding missing tutorial abstracts (#418).

mjpost commented:
Okay, this is done and ready for a quick review. It adds tutorial abstracts from ACL 2016, EACL 2017, and EMNLP 2014–2018.

akoehn commented:
I'll review it in an hour or so.

davidweichiang commented:
Our ingestion scripts convert straight quotes to curly quotes. I would say that for these abstracts, curly quotes are strongly preferred.

mjpost commented:
- [x] I agree, we should use curly. One of the scripts complained about lack of in an abstract, and I changed them to curly.
- [x] Re: EACL 2017: those two pages are not so much abstracts as huge long descriptions. I left the abstract blank as a result. I guess I could put those in, but they are really huge. I just did this
- [ ] I agree the titles are bad for many of the books. Where we added new ones, I made them look nice ("MAIN CONF: Tutorial Abstracts"). But P17, P18 already existed with the (IMO) bad book titles. So the principle was not to change them. Should I change them?

knmnyn commented:
I think for whole volumes, it’s ok for us to make editorial decisions post
publication to bring them in line with guidelines.

Note that previously we didn’t manufacture DOIs for whole volumes so it’s
less of a concern for past years.

- M

>
>    - I agree the titles are bad for many of the books. Where we added new
>    ones, I made them look nice ("MAIN CONF: Tutorial Abstracts"). But P17, P18
>    already existed with the (IMO) bad book titles. So the principle was not to
>    change them. Should I change them?
>
> —
> You are receiving this because your review was requested.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/acl-org/acl-anthology/pull/463?email_source=notifications&email_token=AABU722R7FVBFKDHFCCTHWTP755XHA5CNFSM4IEBE2CKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD2GPKXQ#issuecomment-512554334>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABU724EJZA6V5K2JJX53Z3P755XHANCNFSM4IEBE2CA>
> .
>
-- 
- M
